dw-list-item.js

```diff
@@ -465,12 +500,13 @@ export class DwListItem extends LitElement {
       return;
     }
 
-    if (!el.disabled) { 
-      el.focus && el.focus();
+    if (el.hasAttribute("disabled") || el.disabled) { 
+      let nextFocusableElement = this._getNextFocusableElement(el);
```

Comment:
@devangdreamworld earlier logic was recursive. I wonder why don't we follow recursive (similar) standard again.

We need to meet in-person to understand requirement, and match this logic against it.